terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/WebDriverOperations.java

```diff
@@ -44,6 +48,15 @@ public void setDefaultTimeoutForImplicitlyWait(
     }
 
     /**
+     * Get the text (display value) set for the specified element.
+     * @param by Identifier to look for elements
+     * @return And returns the text (display value)
+     */
+    public String getText(By by) {
+    	return webDriver.findElement(by).getText();
+    }

```

Comment:
14ee9a57b0711e8803968ec72869f8681538e86d 
I understand `terasoluna-gfw-functionaltest` application policy of using WebDriver's API.
So I changed to directly use the WebDriver's API except for complex logic